packages/lumx-react/src/components/thumbnail/Thumbnail.tsx

```diff
@@ -186,6 +196,7 @@ const Thumbnail: React.FC<ThumbnailProps> = ({
                         src={image}
                         alt={alt}
                         loading={loading}
+                        {...(imgProps || {})}
```

Comment:
Maybe could you write `{...imgProps}` and set default empty value in params ⬆️

Reply:
Unfortunately, that will trigger a snapshot update on all the components that use `Thumbnail` on the product side, thus making this change a breaking one

Reply:
Ok :(